packages/maskbook/src/components/InjectedComponents/PayloadReplacer.tsx

```diff
@@ -12,16 +12,23 @@ const useStyle = makeStyles((theme: Theme) => ({
 
 export interface PayloadReplacerProps {
     payload: string
+    theme: Theme
 }
 
-export function PayloadReplacer({ payload }: PayloadReplacerProps) {
+export const PayloadReplacer = withTheme(function ({ payload, theme }: PayloadReplacerProps) {
     const { t } = useI18N()
     const classes = useStyle()
     return (
         <Typography className={classes.text} color="textPrimary" component="span" variant="body1">
-            <Link href={payload.startsWith('https://maskbook.com') ? payload : 'https://maskbook.com'}>
+            <Link
+                ref={(node) => {
+                    if (node) {
+                        node.style.setProperty('color', theme.palette.primary.main, 'important')
+                    }
+                }}
+                href={payload.startsWith('https://maskbook.com') ? payload : 'https://maskbook.com'}>
```

Comment:
```suggestion
                href={/^https:\/\/mask(book\.com|\.io)/.test(payload) ? payload : 'https://mask.io'}>
```

CC @Jack-Works @Tedko Change domain of payload?

Reply:
How about another PR to replace them all?

Reply:
OK, new pr.